File: projects/quarkus-course/getting-started/src/test/java/tech/donau/course/GreetingResourceTest.java
package tech.donau.course;

import io.quarkus.test.junit.QuarkusTest;
import org.junit.jupiter.api.Test;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.is;
import static org.hamcrest.CoreMatchers.startsWith;

import java.util.UUID;

@QuarkusTest
public class GreetingResourceTest {

    @Test
    public void testHelloEndpoint() {
        given()
          .when().get("/hello")
          .then()
             .statusCode(200)
             .body(is("hello"));
    }
    
    @Test
    public void testHelloWithNameEndpoint() {
    	final String name = UUID.randomUUID().toString();
        given()
          .when().get("/hello/" + name)
          .then()
             .statusCode(200)
             .body(is("Hello " + name + ", your id is 1234"));
    }

}

File: projects/quarkus-course/getting-started-kotlin/src/test/kotlin/tech/donau/course/GreetingResourceTest.kt
package tech.donau.course

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured.given
import org.hamcrest.CoreMatchers.`is`
import org.junit.jupiter.api.Test

@QuarkusTest
class GreetingResourceTest {

    @Test
    fun testHelloEndpoint() {
        given()
          .`when`().get("/C:/Program Files/Git/hello")
          .then()
             .statusCode(200)
             .body(`is`("hello"))
    }

}